.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
```

Comment:
Does this mean when a Dependabot PR is merged, a new patch release will be made?

Reply:
IIUC this is more about what happens in a monorepo: https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#updateinternaldependencies

As such we can probably remove it here to avoid confusion.